requirements.txt

```diff
@@ -8,4 +8,5 @@ pyyaml
 numpy
 spydrnet
 werkzeug
-pylint
\ No newline at end of file
+pylint
+JPype1
```

Comment:
Maybe also just put the rapidwright package here and remove the rapidwright_bin recipe

Reply:
I've updated to use our existing rapidwright code.